mixer/pkg/protobuf/yaml/testdata/all/types.pb.go

```diff
@@ -10391,6 +10391,7 @@ func (m *Outer_Inner) Unmarshal(dAtA []byte) error {
 func skipTypes(dAtA []byte) (n int, err error) {
 	l := len(dAtA)
 	iNdEx := 0
+	depth := 0
```

Comment:
Something went wrong here, I would just revert the changes to the mixer stuff?

Reply:
That was produced by make gen, I assumed because one of the base charts was changed. 

I can revert it, though.

Reply:
I think it must be some different dep you have locally or something?